mutalyzer/entrypoints/mutalyzer.py

```diff
@@ -16,6 +16,15 @@
 from .. import variantchecker
 
 
+class MyEncoder(json.JSONEncoder):
+    def default(self, o):
+        json_object = o.__dict__
+        json_object.update({"hgvs": unicode(o), "weight": o.weight()})
+
+        return json_object
+    #default
+#MyEncoder
```

Comment:
This seems like a bit of a weird trick. In any case, we should follow the pattern from the `json` documentation to call `JSONEncoder.default(self, o)` as a fallback.

https://docs.python.org/2/library/json.html#json.JSONEncoder.default